src/migration/1629190095399-AddAverageBlockSizeAndTransactionPerBlockAndTransactionFeeInLast24HourAndMemPool.ts

```diff
@@ -0,0 +1,42 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddAverageBlockSizeAndTransactionPerBlockAndTransactionFeeInLast24HourAndMemPool1629190095399
+  implements MigrationInterface
+{
+  name =
```

Comment:
NB: wondering why is this global variable needed?

Reply:
This is the name of migration.
This file was created by this command.
`yarn typeorm migration:generate -n`
https://stackoverflow.com/questions/65222981/typeorm-synchronize-in-production